test/e2e/README.md

```diff
@@ -26,19 +26,17 @@ When using `kind`, this test involves three `kind` clusters, so you **need to**
 
 When using three pre-existing OCP clusters, your kubeconfig must include contexts named `kscore` (for the kubeflex hosting cluster), `cluster1` and `cluster2`. The following shows an example listing of adequate contexts.
 
-```bash
```

Comment:
Do not remove the code fencing.

The language tag "bash" is a poor choice here; "console" would be better.